css-brochure-form/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 60rem;
```

Comment:
You shouldn't ever limit the width of the body element, it should always be allowed to fill whatever space is necessary. Instead, you should apply this `width` to the `form` element